### PR TITLE
[TACHYON-1029] Add unit test for InodeDirectory.generateClientFileInfo

### PR DESCRIPTION
**Pull request by aaudiber**

### CODE REVIEW

**Comment by gpang** [comment]
ok to test

File: servers/src/test/java/tachyon/master/file/meta/InodeDirectoryTest.java [hunk 1]
```diff
@@ -183,4 +184,25 @@ public void getChildTest() {
...
+	  String path = "/test/path";
+	  FileInfo info = inodeDirectory.generateClientFileInfo(path);
+	  Assert.assertEquals(inodeDirectory.getId(), info.getFileId());
+	  Assert.assertEquals(inodeDirectory.getName(), info.getName());
+	  Assert.assertEquals(path, info.getPath());
+	  Assert.assertEquals("", info.getUfsPath());
+	  Assert.assertEquals(0, info.getLength());
+	  Assert.assertEquals(0, info.getBlockSizeBytes());
+	  Assert.assertEquals(inodeDirectory.getCreationTimeMs(), info.getCreationTimeMs());
+	  Assert.assertEquals(true, info.isIsComplete());
```
Comment (calvinjia) [inline servers/src/test/java/tachyon/master/file/meta/InodeDirectoryTest.java]: Could you use `assertTrue` for boolean comparisons?

Comment (aaudiber) [inline servers/src/test/java/tachyon/master/file/meta/InodeDirectoryTest.java]: Done

File: servers/src/test/java/tachyon/master/file/meta/InodeDirectoryTest.java [hunk 2]
```diff
@@ -183,4 +184,25 @@ public void getChildTest() {
         System.currentTimeMillis() - start));
   }
 
+  @Test
+  public void generateClientFileInfoTest() {
+	  InodeDirectory inodeDirectory = createInodeDirectory();
```
Comment (gpang) [inline servers/src/test/java/tachyon/master/file/meta/InodeDirectoryTest.java]: In tachyon codebase, we use spaces (not tabs) and it should be indented 2 spaces. Thanks!

**Comment by gpang** [comment]
The tab character is breaking the checkstyle check. After that fix, this looks good! Thanks!

**Comment by gpang** [comment]
line 205 is too long (over 100 characters).

**Comment by yupeng9** [comment]
Jenkins, test this please

**Comment by gpang** [comment]
LGTM

**Pull request merged**